packages/main/src/methods/chat-session.ts

```diff
@@ -75,7 +75,7 @@ export class ChatSession {
    *
    * Fields set in the optional {@link SingleRequestOptions} parameter will
    * take precedence over the {@link RequestOptions} values provided at the
-   * time of the {@link GoogleAIFileManager} initialization.
+   * time of the `GoogleAIFileManager` initialization.
```

Comment:
Why didn't the `@link` work here?

Reply:
api-extractor/documenter doesn't really handle multiple entry points and there's 2 separate entry points from the main package and from the `/server` subpath. ChatSession is under the main entry point and GoogleAIFileManager is in the subpath. We run api-documenter separately on each and the two processes don't know about each other. That's why we have a fork of api-documenter in the firebase-js-sdk repo, we wrote custom code to handle multiple entry points. I don't want to use the fork if possible because it's not getting any of the bug fixes and improvements being made to the main branch of api-documenter.

However, now that I've written all that, I realize this is in ChatSession, and this was probably originally a copy-paste mistake, as GoogleAIFileManager doesn't have anything to do with ChatSession. It looks like all the bad GoogleAIFileManager links were typos in the non-server-only entry point, which shouldn't have any references to GoogleAIFileManager at all. I've fixed them to refer to the correct initialization location (`GoogleGenerativeAI.getGenerativeModel()`).